### PR TITLE
Button

### PR DESCRIPTION
**Pull request by TheJaredWilcurt**
I came up with a workaround for the button a11y linting rule.

* [Details here](https://github.com/maranran/eslint-plugin-vue-a11y/issues/28#issuecomment-632134646)

Also went to fix the new linting warnings from the recently merged components. Found that the `vue/max-attributes-per-line` was making one of the files a lot less readable. So I bumped up the amount of attributes allowed per line. Then went back to some places with few attributes and put them back on one line if it was still readable.

### CODE REVIEW

File: src/components/VMultiselect/Autocomplete.vue [hunk 1]
```diff
@@ -72,7 +72,20 @@ import { mixin as clickaway } from "vue-clickaway"
...
+      default: undefined
+    },
+    search: {
+      type: String,
+      default: undefined
+    },
+    options: {
+      type: Array,
+      default: undefined
+    }
```
Comment (TheJaredWilcurt) [inline src/components/VMultiselect/Autocomplete.vue:87]: not sure of the types or defaults here, or if any are required

File: package.json [hunk 2]
```diff
@@ -90,6 +90,7 @@
     "stylelint": "^13.3.3",
     "stylelint-config-sass-guidelines": "^7.0.0",
     "typescript": "^3.8.3",
+    "vue-clickaway": "^2.2.2",
```
Comment (TheJaredWilcurt) [inline package.json:93]: this was missing, causing `npm run docs:build` to fail

Comment (Stegosource) [reply package.json:93]: What was the error? Because this plugin is not used as far as Im aware.

File: src/components/VAlert/VAlert.vue [hunk 3]
```diff
@@ -9,14 +9,15 @@
...
       <slot />
 
-      <!-- eslint-disable vue-a11y/click-events-have-key-events -->
       <button
         v-if="dismissible"
         :aria-label="dismissLabel"
         :class="['vts-alert__dismiss', classes.dismiss]"
         @click="dismiss"
+        @keydown.prevent.enter=""
+        @keyup.prevent.enter="dismiss"
```
Comment (Stegosource) [inline src/components/VAlert/VAlert.vue:18]: Doesn't this cause the event to fire twice?

File: package.json (hunk 2 above)
Comment (TheJaredWilcurt) [reply package.json:93]: it is used in one of the components as a mixin

File: src/components/VAlert/VAlert.vue (hunk 3 above)
Comment (TheJaredWilcurt) [reply src/components/VAlert/VAlert.vue:18]: No, see here: https://github.com/maranran/eslint-plugin-vue-a11y/issues/28#issuecomment-632134646

File: package.json (hunk 2 above)
Comment (TheJaredWilcurt) [reply package.json:93]: `src/components/VMultiselect/Autocomplete.vue:71`

File: src/components/VAlert/VAlert.vue (hunk 3 above)
Comment (Stegosource) [reply src/components/VAlert/VAlert.vue:18]: LGTM

File: package.json (hunk 2 above)
Comment (Stegosource) [reply package.json:93]: Oh, that makes sense. Yeah, this was another case where I copied over some source code from a different project that I want to keep easily available for whenever I build the component in mind.

Probably not in this next release, but somewhere in the near future I want to create a combobox and possibly an autocomplete component. These already exist in the wild, and I was looking through their source code, but will likely need to change some things. 

Long story short, these components dont actually exist for the library yet. These files are just placeholders.

**Comment by Stegosource** [comment]
Did some more research on this, and the "mouse events need keyboard events" logic should only apply to non-interactive elements. Meaning if you add a click event handler to a div, you should also add tabindex and keyboard events. This makes sense.

However, in Vuetensils, all the click event handlers are attached to buttons, which natively support keyboard navigation and fire the click event with the `enter` and `space` key anyway.

I'm putting in an issue on the eslint plugin, as these errors are actually false positives.

**Comment by Stegosource** [comment]
Talked to some folks on Twitter about this as well. It seems that eslint plugin is outdated and a new one is being maintained by the Vue-a11y team. I've swapped them out, and the lint errors are gone.
